rpc_jobs/irr_role_test.yml

```diff
@@ -0,0 +1,102 @@
+- project:
+    name: 'RPC-IRR-Jobs'
+    # Note: branch is the branch for periodics to build
+    #       branches is the branch pattern to match for PR Jobs.
+    series:
+      - master:
+          branch: master
+          branches: "master"
+    image:
+      - xenial:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+      - trusty:
+          IMAGE: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
+    context:
```

Comment:
Yeah, there are plans to add a ceph context at some point, and perhaps others. This serves as the 'scenario' to test.